bullet_train/bullet_train.py

```diff
@@ -98,6 +103,58 @@ def get_value(self, feature_name, identity=None):
         else:
             return None
 
+    def get_trait(self, trait_key, identity):
+        """
+        Get value of given trait for the identity of an environment.
+
+        :param trait_key: key of trait to determine value of (must match 'ID' on bullet-train.io)
+        :param identity: application's unique identifier for the user to check feature state
+        :return: Trait value. None otherwise.
+        """
+        if not trait_key:
+            return None
+
+        if not identity:
+            return None
+
+        data = self._get_flags_response(None, identity)
```

Comment:
Probably more readable here if you use key word args, it might even be required to do so?

e.g.
```data = self._get_flags_response(identity=identity, feature=None)```